### PR TITLE
feat: add nginx rewrites

### LINKED ISSUE #5578: nodejs.org/calendar isn't redirecting to google calendar anymore
Hey!

Very often we share https://nodejs.org/calendar with the community so they keep posted with Node.js meetings. However, this link isn't redirecting to the google calendar anymore

### PR DESCRIPTION
**Pull request by ovflowd**
## Description

This PR introduces the missing rewrites that we've forgot to add before finishing the migration to Vercel. These were sourced from https://github.com/nodejs/build/blob/main/ansible/www-standalone/resources/config/nodejs.org?plain=1

## Validation

Manual Testing that the Rewrites Work

## Related Issues

Fixes https://github.com/nodejs/nodejs.org/issues/5578

### CODE REVIEW

File: next.config.mjs [hunk 1]
```diff
@@ -1,9 +1,12 @@
 'use strict';
 
 import * as nextConstants from './next.constants.mjs';
+import nextRewrites from './next.rewrites.mjs';
```
Comment (AugustinMauroy) [inline next.config.mjs]: what do you thinks about putting all file like this in an folder like "next-utils" ?

**Comment by ovflowd** [comment]
FYI requesting fast-tracking here! cc @nodejs/website as these rewrites are broken right now,

File: next.config.mjs (hunk 1 above)
Comment (ovflowd) [reply next.config.mjs]: They're not utils.

**Review by joesepi — APPROVED** [review]

**Review by rodion-arr — APPROVED** [review]

File: next.config.mjs (hunk 1 above)
Comment (AugustinMauroy) [reply next.config.mjs]: 👍🏼

**Review by araujogui — APPROVED** [review]

**Review by bmuenzenmeyer — APPROVED** [review]

**Pull request merged**